apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/iverksetting/utbetaling/Utbetalingsvedtak.kt

```diff
@@ -50,7 +56,9 @@ enum class Saktype {
     }
 }
 
-data class Attestasjon(val attestant: String)
+data class Attestasjon(
+    val attestant: String
```

Comment:
Du har også enhet her, attestant er vel kun navn.

Reply:
La til enhet for attestant også, slik at vi har sporing på begge i utbetaling.